sdk/translation/azure-ai-translation-document/tests/test_model_updates.py

```diff
@@ -0,0 +1,253 @@
+    def test_supported_formats(self, **kwargs):
+        client = kwargs.pop("client")
+        
+        # get supported glossary formats
+        supported_glossary_formats = client._get_supported_formats(type=FileFormatType.GLOSSARY)
+        assert supported_glossary_formats is not None
+        # validate
+        for glossary_format in supported_glossary_formats.value:
+            self._validate_format(glossary_format)
+
+        # get supported document formats
+        supported_document_formats = client._get_supported_formats(type=FileFormatType.DOCUMENT)
+        assert supported_document_formats is not None
+        # validate
+        for document_format in supported_document_formats.value:
+            self._validate_format(document_format)
+
+    @DocumentTranslationPreparer()
+    @DocumentTranslationClientPreparer()
+    @recorded_by_proxy
+    def test_document_translation_input_args(self, **kwargs):
+        # Creating an instance using required positional arguments
+        source_container_url = "https://t7d8641d8f25ec940prim.blob.core.windows.net/source-12345"
+        target_container_url = "https://t7d8641d8f25ec940prim.blob.core.windows.net/target-67890"
+        doc_input_positional = DocumentTranslationInput(
+                source_url=source_container_url,
+                targets=[TranslationTarget(target_url=target_container_url, language="fr")]
+        )
+        assert doc_input_positional is not None
+        assert doc_input_positional.source_url is not None
+        assert doc_input_positional.targets and doc_input_positional.targets[0].target_url and doc_input_positional.targets[0].language is not None
+
+        # Using keyword-only arguments to specify additional optional parameters
+        doc_input_keyword = DocumentTranslationInput(
+            source_container_url,
+            [TranslationTarget(target_url=target_container_url, language="fr")],
+            source_language="en",
+            storage_type="FOLDER",
+            storage_source="AzureBlob",
+            prefix="start_",
+            suffix="_end"
+        )
+        self.validate_document_translation(doc_input_keyword)
+
+        # Creating an instance using a dictionary to pass parameters
+        params = {
+            "source_url": source_container_url,
+            "targets": [TranslationTarget(target_url=target_container_url, language="fr")],
+            "source_language": "en",
+            "storage_type": "FOLDER",
+            "storage_source": "AzureBlob",
+            "prefix": "start_",
+            "suffix": "_end"
+        }
+        doc_input_dict = DocumentTranslationInput(**params)
+        self.validate_document_translation(doc_input_dict)
+
+    @DocumentTranslationPreparer()
+    @DocumentTranslationClientPreparer()
+    @recorded_by_proxy
+    def test_translation_target_args(self, **kwargs): 
+        # Using keyword arguments to specify additional optional parameters
+        target_keyword = TranslationTarget(
```

Comment:
Do we have a test for testing required positional params for TranslationTarget?

Reply:
Added